pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -31,8 +31,8 @@ import (
 
 // PodAutoscaler is a Knative abstraction that encapsulates the interface by which Knative
 // components instantiate autoscalers.  This definition is an abstraction that may be backed
-// by multiple definitions.  For more information, see the Knative Pluggability presentation:
-// https://docs.google.com/presentation/d/10KWynvAJYuOEWy69VBa6bHJVCqIsz1TNdEKosNvcpPY/edit
+// by multiple definitions. For more information on Knative Autoscalers and Pluggability,
+// [see the Knative Pluggability presentation.](https://docs.google.com/presentation/d/10KWynvAJYuOEWy69VBa6bHJVCqIsz1TNdEKosNvcpPY/edit)
```

Comment:
cc @dprotaso @mattmoor

Reply:
Yeah, I guess we should do some post-processing when generating docs from these and detect links to mark them up with markdown, rather than doing that in Go code and thus breaking godoc.

Reply:
well the issue I am trying to fix here (https://github.com/knative/docs/issues/3143) refers to the fact that we do not have good link previews. 

If we were to detect links and mark them up with markdown, how would we let that post-process know what text would be displayed? i.e. how do we fill in [this part](of the link)

Reply:
Maybe we should drop the link to this presentation anyway. Links to google docs in code feel a little... off.

Reply:
I am down to drop this link